seed/index.js

```diff
@@ -0,0 +1,55 @@
+// const House = require('../db/Schemas/Listing.js');
+
+
+//loop 100 times
+const seed = () => {
+    const houses = [];
+    for (let i = 0; i < 100; i++) {
+        const house = {};
+        let pad = i;
+
+        if (pad.toString().length < 2) {
```

Comment:
I really like what you did here and might use it myself

Reply:
:) i found a good stack overflow post on it